### ISSUE TITLE
[Feature] [Mobile] Don't show keyboard when opening payee/category/account selection modals

### ISSUE THREAD

**Issue by joel-jeremy** [actualbudget/actual#1609]
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

Opening the payee/category/account modals automatically show the keyboard and take up so much screen space that makes it hard to browse through the items in the modals.

### Describe your ideal solution to this problem

Do not open the keyboard automatically when the payee/category/account selections modals are opened.

**Comment by Kidglove57** [comment]
Might the intrusiveness of the keyboard be mobile phone model specific?

I only ask because I do not find it at all intrusive on my iPhone 13 Pro. It only occupies the bottom quarter of the screen and enables me to type into the search box to quickly find the payee or category I need.

**Comment by Cldfire** [comment]
Yeah, the search field is automatically focused (therefore automatically showing the keyboard) on purpose to make it as quick as possible to search for a specific value and then press "enter" on the keyboard to choose it.
